src/dotnet/APIView/APIViewWeb/Models/UsageSampleFileModel.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace APIViewWeb
+{
+    public class UsageSampleFileModel
```

Comment:
I think we should rename this class to UsageSampleRevisionModel and include few additional properties.

1. FileId
2. OriginalFileName
3. CreatedBy
4. CreatedOn
5. RevisionTitle
6. RevisionNumber